tests/Fixer/Comment/NoEmptyCommentFixerTest.php

```diff
@@ -66,10 +66,12 @@ public function provideFixCases()
             ),
             array(
                 '<?php
+                echo 2;
```

Comment:
in theory; sure :) 
however this changes does not remove a test case, it updates a duplicate to a better one.
I simply don't have the time to create another PR for a small case like this on 1.x, even more since I know it is not needed. I put it on my list of todo's, but I don't see this a stopper for this pr.